src/luasocket.h

```diff
@@ -10,8 +10,8 @@
 /*-------------------------------------------------------------------------* \
 * Current socket library version
 \*-------------------------------------------------------------------------*/
-#define LUASOCKET_VERSION    "LuaSocket 3.0-rc1"
-#define LUASOCKET_COPYRIGHT  "Copyright (C) 1999-2013 Diego Nehab"
+#define LUASOCKET_VERSION    "LuaSocket 3.0.0"
+#define LUASOCKET_COPYRIGHT  "Copyright (C) 1999-2022 Diego Nehab"
```

Comment:
The last commit from Diego is in 2019.
We should probably add something more like "luasocket contributors"?

Reply:
Okay, I guess it might be better to just address the version here and to move the copyright to a separate issue?

Reply:
There are a number of other places the license and copyright need updating. Let me take care of than in a dedicated PR, lets just cleanup the version stuff here (still puzzled how I missed thise with all the grepping I did for references to the rc, but hey...)